src/test/scala/cognite/spark/v1/AssetsRelationTest.scala

```diff
@@ -391,14 +391,15 @@ class AssetsRelationTest extends FlatSpec with Matchers with SparkTest {
         .write
         .insertInto("destinationAssetsUpsert")
 
+      val assetsCreated = getNumberOfRowsCreated(metricsPrefix, "assets")
```

Comment:
Do you have any idea why/if moving this check here makes a difference?

Reply:
It does not fix the test, just asserts the easy to  assert and easy to debug stuff first.